<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraS </string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Icons/Resource/camera.png</normaloff>:/Icons/Resource/camera.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>391</width>
        <height>311</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>391</width>
         <height>311</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lblImage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCameraViewfinder" name="cameraWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>391</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Effects</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="7" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="chkInvert">
              <property name="whatsThis">
               <string>Invert Image Colours</string>
              </property>
              <property name="text">
               <string>Invert Colours</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkMirror">
              <property name="whatsThis">
               <string>Mirrot Image</string>
              </property>
              <property name="text">
               <string>Mirror Image</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+F</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkGray">
              <property name="toolTip">
               <string>Gray Scale Image</string>
              </property>
              <property name="text">
               <string>Gray Scale</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+G</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QSlider" name="redSlider">
            <property name="toolTipDuration">
             <number>2</number>
            </property>
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="brightnessSlider">
            <property name="minimumSize">
             <size>
              <width>151</width>
              <height>19</height>
             </size>
            </property>
            <property name="minimum">
             <number>-20</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="sliderPosition">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="lblBlue">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderBlue">
              <property name="minimum">
               <number>-100</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="sliderPosition">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblBright">
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Brightness</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lblGreen">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="greenSlider">
              <property name="minimum">
               <number>-100</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="whatsThis">
    <string>Delay the image cature</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionCapture"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="actionDelay"/>
   <addaction name="actionMail"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/save.png</normaloff>:/Icons/Resource/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Current Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/stop.png</normaloff>:/Icons/Resource/stop.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/folder_open.png</normaloff>:/Icons/Resource/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/camera.png</normaloff>:/Icons/Resource/camera.png</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Capture Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDelay">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/timer.png</normaloff>:/Icons/Resource/timer.png</iconset>
   </property>
   <property name="text">
    <string>delay</string>
   </property>
   <property name="toolTip">
    <string>Delay Capture</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>Resource/Undo-icon.png</normaloff>Resource/Undo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo Last Image Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/Redo-icon.png</normaloff>:/Icons/Resource/Redo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionMail">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Resource/mail.png</normaloff>:/Icons/Resource/mail.png</iconset>
   </property>
   <property name="text">
    <string>mail</string>
   </property>
   <property name="toolTip">
    <string>EMail an image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
